client/src/components/search-bar/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC, useState, Dispatch, SetStateAction, ChangeEvent, FormEvent, useRef } from 'react';
+import styled from 'lib/woowahan-components';
+
+import searchIcon from 'assets/icons/search.png';
+
+import useOutSideClick from 'hooks/use-out-side-click';
+
+import RecentKeywordBox from './recent-keyword-box';
```

Comment:
```suggestion
import React, { FC, useState, Dispatch, SetStateAction, ChangeEvent, FormEvent, useRef } from 'react';
import styled from 'lib/woowahan-components';
import useOutSideClick from 'hooks/use-out-side-click';
import RecentKeywordBox from './recent-keyword-box';
import searchIcon from 'assets/icons/search.png';
```

임포트 문을 조금 정리하면 좋을 것 같아요~

Reply:
* 테스트 코드 작동을 안합니다....
* 임포트 문도 구분을 해서 하는데
  - 모듈 임포트
  - 컴포넌트 임포트
  -  훅 임포트
  - 에셋 임포트
이런 기준이 개인적으로 있어서 이렇게 했는데 여기서는 다 한줄 밖에 없긴 하네요.. 일단 다 붙여놓겠습니다.

Reply:
아 아닙니다 !! 기준이었다면 괜찮습니다 ㅎㅎ